chalice/app.py

```diff
@@ -167,6 +215,10 @@ def __init__(self, view_function, view_name, path, methods,
         #: e.g, '/foo/{bar}/{baz}/qux -> ['bar', 'baz']
         self.view_args = self._parse_view_args()
         self.content_types = content_types
+        if cors is True:
```

Comment:
Yea I was considering pulling it out into a _configure_cors and put a comment in there. I'll just do the comment.